test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go

```diff
@@ -162,3 +228,52 @@ func isFileExistCmd(absoluteFileName string) []string {
 		fmt.Sprintf("if [[ -s %s ]]; then echo true; else echo false; fi", absoluteFileName),
 	}
 }
+
+func createPod(ctx context.Context, c client.Client, ns string, opts ...func(pod *corev1.Pod)) (*corev1.Pod, error) {
```

Comment:
I see this a smart generic solution that deserves to be more public so it can be used also outside this package. maybe the below is a good place:
https://github.com/openshift/cluster-node-tuning-operator/blob/a3b866e0ec5576a8278fa1329e146494a7a4b8e9/test/e2e/performanceprofile/functests/utils/pods/pods.go

Reply:
We might, but I suggest doing that in a separate PR because if we want it to be an "offical" API for creating tests pods in our functests, it requires more attention and additional discussion.